### DISCUSSION TITLE
tkinker program run exe flash cmd window

### DISCUSSION THREAD

**Discussion by jeffxjh** [pyinstaller/pyinstaller#6684] [PyInstaller]
spec file set console=False 
package cmmder is  pyinstaller -F --onefile --clean -n xxx -w .\main.spec.
how can i remove the open exe cmd window?

**Comment by bwoodsend** [comment]
Setting `console=False` should be all you need. If that's not working for you then you'll have to share what you're doing so that we can see what you're doing wrong.

**Reply by jeffxjh** [reply]
i found problems caused by tkinter.iconbitmap() method
